libs/core/src/lib/pagination/pagination.component.html

```diff
@@ -28,7 +28,7 @@
                         (keypress)="onKeypressHandler(page, $event)"
                         (click)="goToPage(page, $event)"
                         *ngIf="page !== -1; else more"
-                        [attr.aria-current]="currentPage == page"
+                        [attr.aria-current]="currentPage === page ? 'page': undefined"
```

Comment:
Shouldn't we use `location` here? 

https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaCurrent#value

Reply:
hmm.. although they say `Represents the current location, for example the current page in a breadcrumbs hierarchy` for `location`, but if you see [this w3 link](https://www.w3.org/TR/wai-aria-1.1/#aria-current), they say `A page token used to indicate a link within a set of pagination links, where the link is visually styled to represent the currently-displayed page.` which is what i followed for `aria-current` value since it even mentioned "**pagination**". For `location`, w3 says that it is `used to indicate the image that is visually highlighted as the current component of a flow chart.`

Reply:
On the W3 site a little below they propose second meaning `Represents the current location within an environment or context`.

There is also great examples here https://www.aditus.io/aria/aria-current They say that 'page' is used to mark an active page from the main menu and 'location` to mark an active nested page.

<img width="359" alt="Screenshot 2021-08-10 at 10 28 55" src="https://user-images.githubusercontent.com/20265336/128818978-105d4e62-a2d5-450b-b8b0-e28b0bdab450.png">

Reply:
The way we use it in pagination is not by having nested links/pages. We are not expecting a pagination scenario with 1>2>3>4 nested pages; for us here each page is independent of the other in pagination component. 
- I tried using `aria-current="location"` instead of `page` but JAWS will read it as `Page x, current location link`. Note that similar to `page` it does not announce on each page switch `Page x, current location link` and only announces the first time(as described in the PR description). 
- There is another [example here](https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-pagination) showing a pagination component using `aria-current="page"`
- Also, the [example](https://www.service-public.fr/particuliers/vosdroits/N23493) you pointed out is actually using `aria-current="page"` ;) 
<img src="https://user-images.githubusercontent.com/53509521/128849557-a187a961-9895-4f4b-9f72-8297b13ab2cb.png" width="400">